environment-dev.yml

```diff
@@ -13,24 +13,9 @@ dependencies:
   - fastprogress>=0.2
   - h5py>=2.7
   - typing-extensions>=3.7
-  - bokeh>=0.12
-  - coverage>=5.1
-  - python-graphviz
   - ipython>=7.16
-  - nbsphinx>=0.4
-  - nose>=1.3
-  - nose-parameterized>=0.6
-  - numpydoc>=0.9
-  - pre-commit>=2.8.0
   - pytest-cov>=2.5
   - pytest>=3.0
-  - recommonmark>=0.4
-  - seaborn>=0.8
-  - sphinx-autobuild>=0.7
-  - sphinx>=1.5
```

Comment:
is there a reason to have all these documentation-related dependencies in here?

Reply:
Yeah, we can move these to a separate env file, but shouldn't just drop them here.

Reply:
OK - ended up putting them back an, as they don't take up much extra time anyway, all I've removed now are some nose/unittest-related ones (which shouldn't be necessary any more, given that pytest is used?), and some (bokeh and seaborn) which are only used in a handful of notebooks